Build error with debug enabled

Hi,

when i enabled the debug option in the Adafruit_BME680.cpp file with `#define BME680_DEBUG`                                                 the compilation fails with:
`
Adafruit_BME680.cpp: In member function 'bool Adafruit_BME680::endReading()':
Adafruit_BME680.cpp:338:51: error: 'meas_period' was not declared in this scope
`

I guess this happend in 7e5a79b6bc3e1e56d32506f0eb03e5f188887345 which changed the name of the variable from `meas_period` to `remaining_millis`.

--- a/Adafruit_BME680.cpp
+++ b/Adafruit_BME680.cpp
@@ -335,7 +335,7 @@ bool Adafruit_BME680::endReading(void) {
   int remaining_millis = remainingReadingMillis();
   if (remaining_millis > 0) {
 #ifdef BME680_DEBUG
-    Serial.print("Waiting (ms) "); Serial.println(meas_period);
+    Serial.print("Waiting (ms) "); Serial.println(remaining_millis);
 #endif
     delay(static_cast<unsigned int>(remaining_millis) * 2); /* Delay till the measurement is ready */
   }

